modules/objdetect/include/opencv2/objdetect.hpp

```diff
@@ -835,6 +835,7 @@ class CV_EXPORTS_W QRCodeDetector
             OutputArrayOfArrays straight_qrcode = noArray()
     ) const;
 
+    CV_WRAP std::vector<std::vector<Point2f> > getQRCorners();
```

Comment:
It's ugly solution. Let's return refined corners with `detectAndDecode` and `detectAndDecodeMulti` as optional output.